bin/opx-config-global-switch

```diff
@@ -15,56 +15,72 @@
 # permissions and limitations under the License.
 
 import argparse
-import sys
```

Comment:
sys is used throughout this file. Why was it removed?

Reply:
It wouldn't have hurt to leave it in, but since everything is being imported from opx_tools as:

`from opx_tools.opx_config_utils import *`

and opx_config_utils.py has `import sys` it is taken care of there. I suppose it does introduce a dependency on the import from opx_config_utils.py though. Do you want me to readd it?

Reply:
I would prefer to see a direct import when I a library is used - helps with the code clarity and avoid breakage in the future when someone removes a reference in one file and breaks the world.

Reply:
That's fair and makes sense. Code updated. Confession: I saw it was grayed out in PyCharm and just deleted it during cleanup 😅. If I had thought about it deliberately my logic would have been the same hahaha. I let the tool drive me instead of the other way around. Whoops.